### DISCUSSION TITLE
GUI wrapper for writing reviews for cargo-crev

### DISCUSSION THREAD

**Discussion by bestia-dev** [crev-dev/cargo-crev#440] [Show and tell]
I announce my first version of the GUI wrapper for writing reviews for cargo-crev.
It is a Linux CLI app that opens the default browser for user input.
On windows, it can be used inside WSL2.
A long description is here:
https://crates.io/crates/cargo_crev_reviews
https://github.com/lucianobestia/cargo_crev_reviews_workspace

It is a similar project to the public list of crev reviews on:  https://web.crev.dev/rust-reviews/.

**Comment by matthiasdebernardini** [comment]
egui would be a good candidate to do this with
